src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
It's better to use `<img>` tag for images wich are part of content

[Style guide](https://mate-academy.github.io/style-guides/htmlcss.html#img-or-background-image)